index.bs

```diff
@@ -561,6 +569,38 @@ It must act as follows:
   </pre>
 </div>
 
+### The {{Blob/stream()}} method ### {#stream-method-algo}
+
+The <dfn method for=Blob>stream()</dfn> method, when invoked, must return
+the result of calling [=get stream=] on the [=context object=].
+
+### The {{Blob/text()}} method ### {#text-method-algo}
+
+The <dfn method for=Blob>text()</dfn> method, when invoked, must run these steps:
+
+1. Let |stream| be the result of calling [=get stream=] on the [=context object=].
+1. Let |reader| be the result of [=get a reader|getting a reader=] from |stream|.
+   If that threw an exception, return a new promise rejected with that exception.
+1. Let |promise| be the result of [=read all bytes|reading all bytes=] from |stream| with |reader|.
+1. Return the result of transforming |promise| by a fulfillment handler that returns the result of
+   running [=UTF-8 decode=] on its first argument.
+
+Note: This is different from the behavior of {{FileReader/readAsText()}} to align better
+with the behavior of {{Body/text()|Body.text()}}. Specifically this method will always
```

Comment:
I'd make the link text something like "Fetch's text()" for clarity. Body is just a specification mechanism.